examples/implementation/platform_independent/SDF_example_002/actor_templates.h

```diff
@@ -0,0 +1,95 @@
+
+#ifndef ACTOR_TEMPLATES_H
+#define ACTOR_TEMPLATES_H
+
+#include "common.h"
+
+struct buffer_nonblocking;
+struct token;
```

Comment:
Should these really be in this file? Shouldn't these already be defined by the headers included in `common.h`?